bin/near-cli.js

```diff
@@ -170,6 +171,22 @@ yargs // eslint-disable-line
         desc: 'Unique identifier for the account',
         type: 'string',
     })
+
+    .option('walletUrl', {
+        desc: 'Website for NEAR Wallet',
+        type: 'string',
+        hidden: true
+    })
+    .option('contractName', {
+        desc: 'Account name of contract',
+        type: 'string',
+        hidden: true
+    })
+    .option('masterAccount', {
+        desc: 'Master account used when create new accounts',
+        type: 'string',
+        hidden: true
```

Comment:
Why hide all of these?

Reply:
This is a good debate to be had. The short answer is, there's a long road to getting `near-shell` consistent, and this is something of a bandaid. Actually, this whole PR is a bandaid.
Note that when you run: `near` to display the default help from yargs, it spits out commands and then the `Options:` section. These are flags that **typically** (again, we're working on it) apply to the top-level commands. Actually, the only good example of this is `--accountId` here.
Then, when you run `near create_account --help` you'll see the relevant `Options:` for that particular command, including `--masterAccount`.
This `hidden` key is not hiding it everywhere, but hiding it from the top-level list. I'm not against removing the `hidden` part, but it'll just make th

Reply:
whoops. It'll just make that a bit longer. The fundamental issue here is we want to use `strict` mode, but we can't do that when we load `config.js` and it contains additional flags that aren't captured at the top level. So in this PR we're like, "yeah, you **can** give me a `masterAccount` flag here, it won't do much on its own, but you can"
And from there `yargs`'s `strict` mode is possible whereby it'll scream at you if you misspell something or provide suggestions (`recommendCommands`) or tell you what you're missing.

Reply:
Excellent explanation, thank you